src/backend/distributed/planner/multi_explain.c

```diff
@@ -197,9 +197,7 @@ CitusExplainScan(CustomScanState *node, List *ancestors, struct ExplainState *es
 
 	if (!ExplainDistributedQueries)
 	{
-		appendStringInfoSpaces(es->str, es->indent * 2);
-		appendStringInfo(es->str, "explain statements for distributed queries ");
-		appendStringInfo(es->str, "are not enabled\n");
+		ExplainPropertyText("ExplainDistributedQueries", "explain statements for distributed queries are not enabled", es);
```

Comment:
I think this key name is not very. I think something better would probably be like this, which mentions the GUC name that is used to control this:

```suggestion
		ExplainPropertyBool("citus.explain_distributed_queries", false, es);
```

Reply:
Thank you for suggestion. Changed commit according to it.